mlrun/serving/routers.py

```diff
@@ -682,14 +700,15 @@ def _parallel_run(self, event, mode: str = ParallelRunnerModes.thread):
             )
         return results
 
-    def validate(self, request):
+    def validate(self, request, method):
```

Comment:
adding a new attribute to a method is risky backward-compatibility-wise. If you are sure `validate` is not used elsewhere its ok. Otherwise add a default value to it. Also, I see the doc string here is NumPy and not ours (reStructuredText), please change it and add type hints.

Reply:
is not used elsewhere.